core/kazoo_perf/src/kz_metrics.erl

```diff
@@ -0,0 +1,118 @@
+%%
+%% Simple (and mostly naive) script that connects to a remote Erlang node,
+%% fetches memory and process statistics and prints them in a Sensu compatible way
+%% for metrics gathering (great for Graphite)
+%%
+-module(kz_metrics).
```

Comment:
@fenollp modules for sup should be _maintenance... please rename to kazoo_pref_maintenance.  Thanks!

For clarity I think the functions should also be renamed to:
graphite_metrics
json_metrics